File: docs/vs-2015/snippets/csharp/VS_Snippets_VSSDK/solutiontoolbar/cs/guids.cs
// Guids.cs
// MUST match guids.h
using System;

namespace Microsoft.SolutionToolbar
{
    static class GuidList
    {
        public const string guidSolutionToolbarPkgString = "400c36a6-c0a9-41dc-9dac-62a237bde6c8";
        public const string guidSolutionToolbarCmdSetString = "fb8bdc2d-a767-4494-817e-06bb8f1e7c92";

        public static readonly Guid guidSolutionToolbarCmdSet = new Guid(guidSolutionToolbarCmdSetString);
    };
}

File: docs/vs-2015/snippets/visualbasic/VS_Snippets_VSSDK/solutiontoolbar/vb/guids.vb
'-----------------------------------------------------------------------------
' Copyright (c) Microsoft Corporation.  All rights reserved.
'-----------------------------------------------------------------------------
' Guids.vb
' MUST match guids.h

Namespace Microsoft.SolutionToolbar
	Friend NotInheritable Class GuidList
		Public Const guidSolutionToolbarPkgString As String = "67eeb039-3775-45ce-aeb5-b2f5c63ef529"
		Public Const guidSolutionToolbarCmdSetString As String = "fc29ba47-6498-4fb8-bea7-88c7092df977"

		Public Shared ReadOnly guidSolutionToolbarCmdSet As New Guid(guidSolutionToolbarCmdSetString)
	End Class
End Namespace